.evergreen/csfle/azurekms/create-and-setup-vm.sh

```diff
@@ -26,8 +26,19 @@ fi
 
 # Set defaults.
 export AZUREKMS_IMAGE=${AZUREKMS_IMAGE:-"Debian:debian-11:11:0.20221020.1174"}
-# Install az.
-"$AZUREKMS_DRIVERS_TOOLS"/.evergreen/csfle/azurekms/install-az.sh
+
+# Check for Azure Command-Line Interface (`az`) version 2.25.0 or newer.
+if ! command -v az &> /dev/null; then
+    echo "az not detected. See https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/azurekms/README.md for supported distros"
+    exit 1
+fi
+AZ_VERSION=$(az version -o json | python -c "import sys, json; print(json.load(sys.stdin)['azure-cli'])" | tr -d '\r\n')
+HAS_REQUIRED_AZ=$(python -c "verstr='$AZ_VERSION'; ver=[int(x) for x in verstr.split('.')]; print ('YES' if ver[0] > 2 or (ver[0] == 2 and ver[1] >= 25) else 'NO')" | tr -d '\r\n')
+if [ "$HAS_REQUIRED_AZ" = "NO" ]; then
+    echo "Detected az version $AZ_VERSION but need version >= 2.25.0. See https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/azurekms/README.md for supported distros"
+    exit 1
+fi
```

Comment:
Suggest verifying this use of Python works on all supported distros listed in the `README.md` before merging.

As an alternative to consider, I've been using the following pattern to test version compatibility in Bash (to deliberately avoid using Python; uses only GNU coreutils `sort` and `head` which is available on all distros):
```bash
EXPECTED_VERSION_OLDER="1.2.0"
EXPECTED_VERSION_NEWER="1.4.0"
ACTUAL_VERSION="1.2.3" # Obtain via `az --version`.

# Exit code 0 (success): actual version satisfies requirement.
[[ "$(printf "$ACTUAL_VERSION\n$EXPECTED_VERSION_OLDER\n" | sort -rV | head -n 1)" == "$ACTUAL_VERSION" ]]

# Exit code 1 (failure): actual version does not satisfies requirement.
[[ "$(printf "$ACTUAL_VERSION\n$EXPECTED_VERSION_NEWER\n" | sort -rV | head -n 1)" == "$ACTUAL_VERSION" ]]
```

As a function:
```bash
check_version() {
  declare actual="${1:?}"
  declare expected="${2:?}"

  [[ "$(printf "$actual\n$expected\n" | sort -rV | head -n 1)" == "$actual" ]]
}

check_version "1.2.3" "1.2.0" # Exit code 0.
check_version "1.2.3" "1.4.0" # Exit code 1.
```

Reply:
Thank you for the suggestion. I find the bash approach easier to read than the Python one liner and have updated.

Latest changes verified with these patches:
- C driver: https://spruce.mongodb.com/version/643ab7091e2d1742f32e2762
- Java driver: https://spruce.mongodb.com/version/643ab73d56234382bf2c8bef